test/e2e/tests/onboarding.spec.js

```diff
@@ -19,6 +28,188 @@ describe('MetaMask onboarding', function () {
       },
     ],
   };
+
+  it('Clicks create a new wallet, accepts a secure password, reveals the Secret Recovery Phrase, confirm SRP', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await completeCreateNewWalletOnboardingFlow(
+          driver,
+          testSeedPhrase,
+          testPassword,
+        );
```

Comment:
could we maybe end all the `it` blocks with at least one assertion? For example, in this case, we might expect to land on the Home screen

Reply:
Done